### PR TITLE
Issue template and PR template

### LINKED ISSUE #99: Issue and PR templates
Relating this to the idea of issue and PR template from discussion #92 

Any criteria you guys wish to  have or mandate on the template.
Am currently preparing a template, let me know what you guys have in mind

### PR DESCRIPTION
**Pull request by sreecodeslayer**
### Issue Reference
This PR addresses a suggestion from #92 and Fixes #99 

### Summarize
This adds two files under the hidden `.github` folder, that helps as a guideline for creating Issues and PRs.
Users can follow those templates so that the workflow becomes easier on both ends of development.

> Let me know if there is anything we can add to this.

### CODE REVIEW

**Comment by salmanulfarzy** [comment]
Please see #62

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+### Subject of the issue
+Describe your issue here.
+
+### Your environment
+* version of your python
+* which browser and its version
```
Comment (bobinson) [inline .github/ISSUE_TEMPLATE.md]: OS type (iOS or Android)

**Comment by bobinson** [comment]
@biswaz : please enter this as default as there are too many PRs and it will become uncontrollable.

@sreecodeslayer

**Comment by vrthra** [comment]
This is not really testable, but on review it looks a good suggestion -- marking it as tested.

**Comment by shajith** [comment]
this looks like a good idea!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+### Issue Reference
+This PR addresses the Issue : Fixes #
```
Comment (shajith) [inline .github/PULL_REQUEST_TEMPLATE.md]: Might be a good idea to add a note here to check Github issues for any dupes. Eg:

Note: Please check the Issues on this repo first to make sure you are not duplicating work. Go here: https://github.com/IEEEKeralaSection/rescuekerala/issues

**Pull request merged**